### ISSUE TITLE
[Controller Request]:

### ISSUE THREAD

**Issue by juararob** [ndeadly/MissionControl#626]
### Controller Name

Nacon - MG-X Pro

### Current Controller Behaviour

Controller is not successful in pairing with the console

### Supported Features

- [ ] Rumble/Vibration
- [ ] Motion Controls
- [ ] Player Number Indicator
- [ ] Battery Level Reporting

### Additional Details

https://www.nacongaming.com/es-ES/mando-movil-mg-x-pro

**Comment by ndeadly** [comment]
My guess would be that this controller is using Bluetooth LE, due to not being able to connect it. The website also advertises using Bluetooth 4.x which is often an indicator of this too. If you can pair it with a PC you should be able to check this. In Windows you can find the info in device manager under device properties.

**Comment by juararob** [comment]
Yes... I suppossed you are right! On windows it said Bluetooth LE...

Thanks!

Hope Bluetooth LE come soon!

[image: image.png]
